vulnerabilities/migrations/0035_auto_20221125_1257.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.0.7 on 2022-11-25 12:57
```

Comment:
You need a better migration script name as this is not a generated migration, is it?